### ISSUE TITLE
"Select All" does nothing except select text

### ISSUE THREAD

**Issue by markcarver** [tympanix/Electorrent#130]
**Client:**
- [ ] µTorrent
- [ ] qBittorrent
- [ ] Transmission
- [x] rTorrent

**Client Version:**
3.8 (but not really applicable to this issue)

**Operating System:**
macOS 10.13.6 (17G65) (but likely happens on Windows and Linux too)

**Application Version:**
2.4.0 (2.4.0.291)

**Current behavior:**
When on any of the torrent list views (all, downloading, finished, seeding, stopped, failed), pressing `Cmd+A` (`Ctrl+A` on Windows/Linux) does nothing.

**Expected behavior:**
Pressing `Cmd+A` (`Ctrl+A` on Windows/Linux) would select all the torrents in the current torrent list view.

**Comment by tympanix** [comment]
That is true. The `Select All` functionality is for selecting text. Maybe that is misleading. The functionality you describe would certainly be helpful, as long as `Ctrl+A` would still have its intended purpose in text boxes.

**Comment by markcarver** [comment]
Yes, I would expect it to work for both.

However, given that there is no text (except for in settings), it's completely useless on the main views.

It's also painful when needing to select a group of torrents that need to be stopped or deleted.

From looking the electron code, it's currently just defaulting to the system handler. I imagine the solution would be to implement a custom handler that detects whether there's any textbox currently focused to determine behavior.

**Comment by tympanix** [comment]
To add on to this,  `Shift`and `Ctrl`/`Cmd` works as expected for selecting a group of torrents.
